apps/pie-microsite/src/assets/styles/components/_contentPage.scss

```diff
@@ -61,80 +66,88 @@
 }
 
 .c-contentPage-header-subheading {
```

Comment:
I'm so sorry. Please could we rename this class to `c-contentPage-header-subheading` just so it's consistent with other code?